### ISSUE TITLE
Update Retriever.jl to work on 1.1.0 

### ISSUE THREAD

**Issue by henrykironde** [weecology/retriever.jl#39]
This may require changing project.toml.
Using the latest current version of python. 

**References:** 
https://julialang.github.io/Pkg.jl/v1/managing-packages/

**Artifacts that we shall need to change:**

After testing locally we need to change CI settings to pass for julia 1.1
Current version julia:0.7.0-stretch

Docker file https://github.com/weecology/Retriever.jl/blob/master/Dockerfile
```J
FROM julia:0.7.0-stretch

```
https://github.com/weecology/Retriever.jl/blob/master/Dockerfile

**Backward compatibility:**
Since we are testing with docker, we need to create two docker files as Dockerfile_[version]. One to fire up an image of `julia:0.7.0-stretch` and the other for  `julia 1.1`

**Comment by ethanwhite** [comment]
Regarding backward compatibility - do we have much using on `0.7.0`? My temptation is to keep things simple since we're at `0.0.2` and just drop `0.7` support. For something as leading edge as Julia I'd assume there's been a major move over to 1.0+. Thoughts?

**Comment by ethanwhite** [comment]
Now passing on Julia 1.1.1 on Travis: https://travis-ci.org/ethanwhite/Retriever.jl/builds/547460253

**Comment by ethanwhite** [comment]
Current status:

- [x] Working locally when installing using `include`
- [x] Passing on Travis
- [x] Working when installing using Pkg

When attempting to install using Pkg the installation goes smoothly. Running `using Retriever` works fine. Calling Retriever functions results in the following error:

```julia
julia> Retriever.install_csv("iris")
ERROR: ArgumentError: ref of NULL PyObject
Stacktrace:
 [1] getproperty(::PyCall.PyObject, ::String) at /home/ethan/.julia/packages/PyCall/ttONZ/src/PyCall.jl:303
 [2] getproperty(::PyCall.PyObject, ::Symbol) at /home/ethan/.julia/packages/PyCall/ttONZ/src/PyCall.jl:313
 [3] #install_csv#2(::String, ::String, ::Bool, ::Bool, ::Function, ::String) at /home/ethan/.julia/packages/Retriever/4if63/src/Retriever.jl:80
 [4] install_csv(::String) at /home/ethan/.julia/packages/Retriever/4if63/src/Retriever.jl:80
 [5] top-level scope at none:0
```

**Comment by henrykironde** [comment]
Could be the version of `pycall`

**Comment by ethanwhite** [comment]
Here's the version information:

```julia
(v1.1) pkg> status
    Status `~/.julia/environments/v1.1/Project.toml`
  [35a29f4d] DocumenterTools v0.1.1
  [438e738f] PyCall v1.91.2
  [ed6da631] Retriever v0.0.2 #julia1pt0 (.)
```

**Comment by ethanwhite** [comment]
Opened an issue with PyCall in case they have any ideas: https://github.com/JuliaPy/PyCall.jl/issues/699

**Comment by ethanwhite** [comment]
Implemented in 3400f6a2f7dffffd84f299480b376eac949f03cb.
